docs/content/deployments/server/deploying-on-microtica.md

```diff
@@ -0,0 +1,125 @@
+
+Then, choose whether you want a production Medusa Backend environment or a development one. The production template will provision your managed RDS PostgreSQL and Redis instances. On the other hand, the development template will use a local SQLite and a fake Redis instance.
+
+![Configure Template](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f9d62c7658915a3788_ERA-ETfPfTiCQ8wDPQZX6IxCZiDhX9GIj-jNTxIUfh6AKdC0BjdVKmuJbaaUlzQeITZWW00Ro1vFZj2R_arYBqD1AbWwlygPwzMOlwYGXby9ZtZ4pfI1A7jZ-oI4fmWHA3VmXtB1QR8NERVxEsT9R1zPQujSTIaWtQUs4gMAM6rVvA6iOfQZR7Qx1NcY.png)
+
+### Step 3: Connect an AWS account 
+
+In the last step before deployment, select the environment in which you want to deploy the template. An existing default environment called `development` will be preselected here, or you can create a new environment.  
+
+Then, connect your AWS account when prompted. This process takes only a few seconds, so afterward only choose the region you want to deploy in. 
+
+![Connect AWS account](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f98144a7eb845dabee_GYZFu6wD5huAf6H5sCs4qpz09wQOaI2IMThsEd0CwR4PePc1ryF4mAlb4YkTZdIkX3Mnh_OzEr9bdm2r1_5GkYHHlFgHksP_hDj7DmlnitZmWaotj5th3R4wHK6gJzhIDmjfXeCk30YdK1kaeGH41JRCyteVH9y3YqzIyGz8xQlcDYKCoWVxDey47mPw.png)
+
+### Step 4: Deploy to AWS
+
+Finally, a deployment summary of what will be provisioned on your AWS account is presented. Click on the "Deploy" button to trigger a deployment of the template and start creating the infrastructure for a Medusa backend.
+
+![Deployment Summary](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f932d1b32052287126_h96ejVytOhLLEUCeIWbQu-OG4oxJQqj5z9ZJXjH5EboT5gnw6s61hl2mvHcUaGQUyJMQtNdyb4hdFWDvYlFBa8wznkBus7nkXR_0JAIRvN5QDdZP6WCDNSzYeBSJcoV-ifDdUHzJODF_Oxv6mgIsHtpj92PA6VgQsaYM0n_xyzV-lhalH6kdR-tku2hb.png)
+
+It will take around ten minutes for the solution to be deployed on your cloud account.
+
+You can follow the build pipeline in real-time by clicking the "View Logs" button. 
+
+![Building Resources](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f9a465c27cf7090e5f_gxwdXAAmUxHsSyJlgScHuu4m7GiFTNnwW-OZoAhDC8v-Ez3KmsK-OCKfF2Qe2JUtxMyih7jCA7n-hAZnx0fdFjv-TWyDuoAUvftSRE9hqmPB1gVhG6lH0IhXqbliou1VnAVDvlCtHobzlUrrDhnCmzob9PfdNGrzlhwsL60QsAXJm1gIPE27J6P_Y2Y3.png)
+
+Once the build process is complete, a new deployment with the infrastructure resources is triggered. You can follow the logs of the deployment process by clicking the "View deployment" button, and then selecting the deployment from the list. 
+
+![Deployment Logs](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f93aa3974438e56689_vF-4U1L5B_sQQJdrjaE_bdlN2hdTsOcn5eWVRfgzF1UqNAuWAqN2zh9ttWp6SW2jjCqPUNLpRsL4j01FR5emAMcJqCbfwbaBv3Np49JH6hpggYm3rmY-xYCOg8H0dooKnC4x_sTD-JreveLzUxBMxQI3M0gCm-8B2YJ5eJj7dU8mf8qlSTYmnpnExIvB.png)
+
+---
+
+## Preview the environment
```

Comment:
We recently embedded the admin plugin as part of the medusa backend. I'm not sure if you've tested it, but if it works can you add a section similar to [here](https://docs.medusajs.com/deployments/server/deploying-on-railway#testing-the-admin) on how to test it?

if it doesn't work with the embedded admin then you need to add a note about it as well.

Also change the header to the following:
```suggestion
## Test the Backend
```